stable/index.yaml

```diff
@@ -11,12 +23,12 @@ entries:
     sources:
     - https://github.com/portworx/helm
     urls:
-    - https://raw.githubusercontent.com/portworx/helm/master/stable/portworx-1.0.0.tgz
+    - https://github.com/logdnalf/helm/blob/per_usage_licensing/stable/portworx-1.0.0.tgz
```

Comment:
These need to point to the portworx repo and not the fork

Reply:
Fixed.